libs/verifiable-credentials/src/main/java/uk/gov/di/ipv/core/library/verifiablecredential/helpers/VcHelper.java

```diff
@@ -223,4 +223,13 @@ private static Integer getAge(String dobValue) {
             return null;
         }
     }
+
+    public static Vot getVcVot(SignedJWT vc) {
+        try {
+            return Vot.valueOf(vc.getJWTClaimsSet().getStringClaim(VOT_CLAIM_NAME));
+        } catch (Exception e) {
```

Comment:
Rather than catching all `Exception`s here, it's better to be more specific and just catch the `ParseException` that `getJWTClaimsSet` can thrown.